paasta_itests/steps/marathon_steps.py

```diff
@@ -33,5 +37,28 @@ def create_trivial_marathon_app(context):
 
 @then(u'we should see it running in marathon')
 def list_marathon_apps_has_trivial_app(context):
-    assert 'test--marathon--app' in paasta_tools.marathon_tools.list_all_marathon_app_ids(context.marathon_client)
-    assert context.marathon_client.get_app('/test--marathon--app')
+    actual = paasta_tools.marathon_tools.list_all_marathon_app_ids(context.marathon_client)
+    assert APP_ID in actual
+    assert context.marathon_client.get_app('/%s' % APP_ID)
+
+
+@then(u'it should show up in marathon_services_running_here')
+def marathon_services_running_here_works(context):
+    with mock.patch('paasta_tools.mesos_tools.socket.getfqdn', return_value='mesosslave'):
+        (discovered,) = paasta_tools.marathon_tools.marathon_services_running_here()
+
+    assert discovered[0] == u'test_marathon_app', repr(discovered)
```

Comment:
yeah, I wish that behave had the magical assertion stuff that pytest has. Without this you just get AssertionError.